tsconfig.base.json

```diff
@@ -25,9 +25,6 @@
       "@code-pushup/lighthouse-plugin": [
         "packages/plugin-lighthouse/src/index.ts"
       ],
-      "@code-pushup/lighthouse-plugin/testing": [
-        "packages/plugin-lighthouse/test/index.ts"
-      ],
       "@code-pushup/models": ["packages/models/src/index.ts"],
       "@code-pushup/models/testing": ["packages/models/test/index.ts"],
```

Comment:
This one should also be deleted, I believe:

```suggestion
```

Reply:
Not yet, this PR is part 1 of 2. I still need to clean up all the main config types and move `test` folder to `testing-utils`.